kokoro/scripts/test/go_test.sh

```diff
@@ -53,6 +53,45 @@ track_flakiness
 # Avoids "fatal: detected dubious ownership in repository" errors on Kokoro containers.
 git config --global --add safe.directory "$(pwd)"
 
+# A helper for parsing YAML files.
+# Ex: VALUE=$(yaml ~/my_yaml_file.yaml "['a_key']")
+function yaml() {
+  python3 -c "import yaml;f=yaml.safe_load(open('$1'))$2;print(','.join(str(i) for i in f) if type(f)==list else f);"
+}
+
+function set_platforms() {
+  # if PLATFORMS is defined, do nothing
+  if [ -n "${PLATFORMS}" ]; then
```

Comment:
go/shell-style#tests says to use `[[` instead of `[`.

Reply:
Done